### DISCUSSION TITLE
why does my message alert success have ugly green vs the glorious looking prime sample?

### DISCUSSION THREAD

**Discussion by shougom-hennepin** [primefaces/community#1000] [PrimeNG] [answered]
![image](https://github.com/primefaces/community/assets/109691091/06a29a52-71b1-4ed4-b5f2-03de9eea0e95)

vs 

![image](https://github.com/primefaces/community/assets/109691091/f77aaafc-bf2c-469f-8c50-f1852741726b)

Ive followed the prime ng sample stackblitz to the letter.  I think.  Im using message service.

html
<p-messages></p-messages>

TS
this.messageService.add({severity:'success', summary:'Project record updated!', detail:'This project record was updated successfully'});

**Comment by shougom-hennepin** [comment]
Ok i just realized we are using a custom theme.  Im wondering how i can switch it back with ngDeep or something.  I like primes success style better.

Sass below.  

 &.p-message-success {
        background: $successMessageBg;
        border: $successMessageBorder;
        border-width: $messageBorderWidth;
        color: $successMessageTextColor;

        .p-message-icon {
            color: $successMessageIconColor;
        }

        .p-message-close {
            color: $successMessageIconColor;
        }
    }

**Reply by shougom-hennepin** [reply]
I was able to override it with this in my local css file.  However the icon looks not as nice as the sample.

:host::ng-deep .p-message.p-message-success {
  background: rgba(228, 248, 240, 0.7);
  border: solid #1ea97c;
  border-width: 0 0 0 6px;
  color: #1ea97c;
  }

![image](https://github.com/primefaces/community/assets/109691091/f7ddc963-9023-4da0-8067-0ec1d38fe226)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Could you please share a stackblitz reproducer so we can identify the issue?
